web/src/user_group_pill.ts

```diff
@@ -1,10 +1,29 @@
 import * as user_groups from "./user_groups";
 
-function display_pill(group) {
-    return group.name + ": " + group.members.size + " users";
+type UserGroup = {
+    id: number;
+    name: string;
+    description: string;
+    members: Set<number>;
+    is_system_group: boolean;
+    direct_subgroup_ids: Set<number>;
+};
```

Comment:
Shouldn't this be imported from `user_groups.ts`? I don't see why we'd duplicate a definition like this.

Reply:
thanks for your review, I have updated, the code accordingly